interview-questions.md

```diff
@@ -9,68 +9,82 @@ Then, research the question to expand on your answer. Even if you feel you have
+  <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div>
+  The HTML <span> element is a generic inline container for phrasing content, which does not inherently represent anything. It can be used to group elements for styling purposes (using the class or id attributes), or because they share attribute values, such as lang. It should be used only when no other semantic element is appropriate. <span> is very much like a <div> element, but <div> is a block-level element whereas a <span> is an inline element.
+  <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span>
+
 
 
 
 3. What is a ternary operator in JavaScript?
 
-  Your answer:
+  Your answer: A ***ternary operator*** in javascript allows for a shorten version of checking a if else statement that only accepts a true or false condition. It allows either a true condition code block or a false condition code block to execute. 
 
   Researched answer:
-
-
+  The conditional (ternary) operator is the only JavaScript operator that takes three operands: a condition followed by a question mark (?), then an expression to execute if the condition is truthy followed by a colon (:), and finally the expression to execute if the condition is falsy. This operator is frequently used as a shortcut for the if statement.
+  <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator>
 
 4. What are props in React?
 
-  Your answer:
+  Your answer: ***Props*** in react are variables that are passed down from higher components. Props are usually associated with data that is displayed to the user.
 
   Researched answer:
+  Props” is a special keyword in React, which stands for properties and is being used for passing data from one component to another. But the important part here is that data with props are being passed in a uni-directional flow
+  <https://itnext.io/what-is-props-and-how-to-use-it-in-react-da307f500da0>
 
 
 
 5. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: The ***this*** keyword is a special keyword that references the object that ***this*** keyword is created in. It helps to reference methods and properties and are specifically bind to the object that is binded too. If the ***this*** keyword loses context of their object than it will reference the global window object.x
 
   Researched answer:
+  In most cases, the value of this is determined by how a function is called (runtime binding). It can't be set by assignment during execution, and it may be different each time the function is called. ES5 introduced the bind() method to set the value of a function's this regardless of how it's called, and ES2015 introduced arrow functions which don't provide their own this binding (it retains the this value of the enclosing lexical context).
```

Comment:
Great answer and great Researched answer. This and how it relates to specific objects/ and how you bind those objects is something that comes up frequently in Dev life.